Submitted By: Jeremy Utley (jeremy@linuxfromscratch.org)
Date: 2004-04-11
Initial Package Version: alpha9.8
Upstream Status: Not Submitted
Origin: Self Created
Description: Fixes syntax issues with GCC 3.4

$LastChangedBy: randy $
$Date: 2004-08-10 15:20:53 -0600 (Tue, 10 Aug 2004) $

diff -aur cdparanoia-III-alpha9.8/interface/utils.h cdparanoia-III-alpha9.8-new/interface/utils.h
--- cdparanoia-III-alpha9.8/interface/utils.h	2000-04-19 15:41:04.000000000 -0700
+++ cdparanoia-III-alpha9.8-new/interface/utils.h	2004-04-12 00:10:53.092860574 -0700
@@ -112,6 +112,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+    break;
     }
   }
 }
@@ -127,6 +128,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+    break;
     }
   }
 }
@@ -169,6 +171,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+    break;
     }
   }
   if(malloced)free(buffer);
@@ -205,6 +208,7 @@
       break;
     case CDDA_MESSAGE_FORGETIT:
     default:
+    break;
     }
   }
   if(malloced)free(buffer);
